workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/PurchasedAppsTable.tsx

```diff
@@ -183,9 +183,14 @@ const AppsTable: React.FC<AppsTableProps> = ({items}) => {
 										</DropDown.Item>
 
 										<DropDown.Item
+											disabled={
+												orderTypeExternalReferenceCode !==
+												OrderType.CLOUD
+											}
 											onClick={() => {
-												window.location.href =
-													'https://console.marketplacedemo.liferay.sh/projects';
+												window.open(
+													'https://console.liferay.cloud'
+												);
```

Comment:
We need to leverage CX Properties for that, not having this data hard-coded

Something like this: https://github.com/liferay/liferay-portal/blob/aa4e661cf8d6e38b797edb6617bdba46e2e075e7/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/main.tsx#L22

https://github.com/liferay/liferay-portal/blob/aa4e661cf8d6e38b797edb6617bdba46e2e075e7/modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/layouts/testray/page-definition.json#L49

Reply:
Done!